code/modules/clothing/spacesuits/void/station_vr.dm

```diff
@@ -0,0 +1,42 @@
+//Exploration
+/obj/item/clothing/head/helmet/space/void/exploration
+	name = "exploration voidsuit helmet"
+	desc = "A radiation-resistant helmet made especially for exploring unknown planetary environments."
+	icon_state = "helm_explorer"
+	item_state = "helm_explorer"
+	armor = list(melee = 50, bullet = 15, laser = 25,energy = 35, bomb = 30, bio = 100, rad = 70)
+	light_overlay = "helmet_light_dual" //explorer_light
+
+/obj/item/clothing/suit/space/void/exploration
+	name = "exploration voidsuit"
+	desc = "A lightweight, radiation-resistant voidsuit, featuring the Explorer emblem on its chest plate. Designed for exploring unknown planetary environments."
+	icon_state = "void_explorer"
+	armor = list(melee = 50, bullet = 15, laser = 25,energy = 35, bomb = 30, bio = 100, rad = 70)
```

Comment:
50 seems high for these, given that almost every threat they face will be melee. You're basically giving them a 50% reduction in incoming damage entirely with this. Given the number of "exploration is too easy" issues open, plus the fact that a PR that simply adds armor to combat exactly the threat an entire set of jobs faces is kinda 'eh', this should probably be reduced. 35? Maybe?